<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogUart</class>
 <widget class="QDialog" name="DialogUart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnUartOpen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>openUart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUartClose">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>closeUart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDisConnect">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readAll">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>readAll</string>
   </property>
  </widget>
  <widget class="QPushButton" name="writeAll">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>writeAll</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="readText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="writeText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
